database/tables/pl_courses.pg

```diff
@@ -5,6 +5,7 @@ columns
     grading_queue: text
     id: bigint not null default nextval('pl_courses_id_seq'::regclass)
     institution_id: bigint not null default 1
+    is_example_course: boolean not null default false
```

Comment:
Sorry to suggest this, but we don't use the `is_XXX` pattern for boolean columns anywhere else in the DB, so we shouldn't do so here. I think calling this column just `example_course` would match the pattern we use elsewhere (e.g., `variants.broken`, `assessment_instances.open`).

Reply:
Renamed in 957fc6db94a1c14c43501cd37506b159bb16c1e4